lib/msal-browser/test/cache/TestStorageManager.ts

```diff
@@ -33,6 +38,25 @@ export class TestStorageManager extends CacheManager {
         return null;
     }
 
+    removeAccountKeyFromMap(key: string): void {
+        const currentAccounts = this.getAccountKeys();
+        const removalIndex = currentAccounts.indexOf(key);
+        if (removalIndex > -1) {
+            currentAccounts.splice(removalIndex, 1);
+            this.store[ACCOUNT_KEYS] = currentAccounts;
+        }
```

Comment:
A bit more efficient:
```suggestion
            this.store[ACCOUNT_KEYS] = currentAccounts.filter(entry => entry !== key);
```

Reply:
Ah, missed the fact that this is for tests - not super important then. Up to you

Reply:
Honestly our tests can also use a perf boost. This is minor but gradual improvements are the way to go, thanks!